File: app/src/main/java/maderski/markparkingspot/DropPreviousPinActivity.java
package maderski.markparkingspot;

import android.app.Activity;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class DropPreviousPinActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        MPSNotification notification = new MPSNotification(this);
        notification.createMessage(true, false);
        finish();
    }
}

File: app/src/main/java/maderski/markparkingspot/DropPreviousPinActivity.kt
package maderski.markparkingspot

import android.app.Activity
import android.content.Intent
import android.net.Uri
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class DropPreviousPinActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val pinLabel = "Parking Spot " + MPSPreferences.getCurrentDate(this)
        val latitude = MPSPreferences.getLatitude(this)
        val longitude = MPSPreferences.getLongitude(this)

        val dropPinIntent = Intent(Intent.ACTION_VIEW,
                Uri.parse("geo:0,0?q=$latitude,$longitude($pinLabel)"))
        startActivity(dropPinIntent)
        finish()
    }
}
